src/main/scala/org/deepdive/ddlog/DeepDiveLogCompiler.scala

```diff
@@ -346,7 +346,7 @@ class QueryCompiler(cq : ConjunctiveQuery, ss: CompilationState) {
       case _ => s" AS column_${cq.headTerms indexOf e}"
     }
     case ViewAlias => s" AS column_${cq.headTerms indexOf e}"
-    case UseVariableAsAlias => s""" AS "${DeepDiveLogPrettyPrinter.print(e)}""""
+    case UseVariableAsAlias => s""" AS "${DeepDiveLogPrettyPrinter.print(e)}"""" //"
```

Comment:
This is for the editor syntax highlighting right?  What about using `""" "` instead?